charts/probate-frontend/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Helm chart for the HMCTS probate product
 name: probate-frontend
 home: https://github.com/hmcts/probate-frontend
-version: 2.0.12
+version: 2.0.13
```

Comment:
Make any changes to the `/charts` folder in a separate PR, get that approved and merged first.
This is to avoid race conditions in Flux.